site/content/docs/5.3/components/navbar.md

```diff
@@ -52,13 +62,19 @@ Here's an example of all the sub-components included in a responsive light-theme
           <a class="nav-link" href="#">Link</a>
         </li>
         <li class="nav-item dropdown">
-          <a class="nav-link dropdown-toggle" href="#" role="button" data-bs-toggle="dropdown" aria-expanded="false">
+          <a
```

Comment:
Most of the formatting proposed in this PR is not compatible with what we already have in place. Please remove all the auto-formatting.

Reply:
Apologies if I caused any inconvenience, I will do better in future contributions, apologies once again

Reply:
Don't worry, no need for apologies :)